app/presenters/single_content_item_presenter.rb

```diff
@@ -65,6 +65,18 @@ def feedex_context
     I18n.t("metrics.feedex.context")
   end
 
+  def feedback_explorer_href
+    template = Addressable::Template.new(
+      "#{Plek.find('support')}/anonymous_feedback{?from,to,paths}"
```

Comment:
I think here, `Plek.new.external_url_for`is the thing to use, as that will generate the right URL for the Support app in AWS environments which use different URLs for internal and external traffic.

Reply:
Yup! Changed to `external_url_for`